ansible/roles_ocp_workloads/ocp4_workload_rhods_olm/tasks/workload.yml

```diff
@@ -0,0 +1,170 @@
+---
+# vim: set ft=ansible
+# Implement your Workload deployment tasks here
+- name: Create the projects
+  k8s:
+    state: present
+    kind: Namespace
+    name: "{{ item }}"
+  loop: "{{ ocp4_workload_rhods_olm_namespace_list }}"
+
+- name: Create the modh puller secret in various namespaces
+  k8s:
+    state: present
+    namespace: "{{ item }}"
+    definition: "{{ lookup('template', './templates/pullsecret.yaml.j2' ) | from_yaml }}"
+  loop: "{{ ocp4_workload_rhods_olm_namespace_list }}"
+
+- name: Create the modh puller secret in openshift-marketplace
+  k8s:
+    state: present
+    namespace: "openshift-marketplace"
+    definition: "{{ lookup('template', './templates/pullsecret.yaml.j2' ) | from_yaml }}"
+
+- name: Create RHODS Pagerduty Secret
+  k8s:
+    state: present
+    definition: "{{ lookup('template', 'pagerduty-secret.yaml.j2' ) }}"
+
+- name: Create CatalogSource for use with catalog snapshot
```

Comment:
Seems that the `install_operator` role (that I wrote because you asked for it :-) ) could be used instead of duplicating the logic.